bl/tx/dbtx.go

```diff
@@ -23,6 +23,12 @@ type Store struct {
 	db *sql.DB
 }
 
+func NewStore(db *sql.DB) Transactioner {
```

Comment:
I'm not sure whether Transactioner is a good name for the interface. I think sql package already has that interface and this looks controversial. I'd suggest you can interface Store and you can call the struct TransactionStore or TxStore